src/Controller/SiteInfoController.php

```diff
@@ -169,6 +347,7 @@ protected function getBundlesByNumberOfFields(array $data) {
   public function siteInfo(Request $request) {
     $rule = $request->query->get('rule', 'count');
     $index = (int) $request->query->get('index', '0');
+    $percent_of_instances_threshold = (float) $request->query->get('percent_of_instances_threshold', '100.1');
```

Comment:
It's because of BC. If we set the default to 100%, then we will get additional fields. With `>100%` value we will get only required fields and current tests in distribution will continue to work properly.

We can change it when we start to use  `percent_of_instances_threshold` in Nightwatch tests.